src/logging.c

```diff
@@ -87,3 +97,46 @@ void log_sensors_distance(void)
 
 	LOG_INFO("%f,%f,%f,%f", sl_dist, sr_dist, fl_dist, fr_dist);
 }
+
+/**
+ * @brief Log all sensors raw readings.
+ */
+void log_sensors_raw(void)
+{
+	uint16_t off[NUM_SENSOR];
+	uint16_t on[NUM_SENSOR];
+
+	get_sensors_raw(off, on);
+
+	LOG_INFO("OFF-ON,%d,%d,%d,%d,%d,%d,%d,%d", off[SENSOR_SIDE_LEFT_ID],
+		 off[SENSOR_SIDE_RIGHT_ID], off[SENSOR_FRONT_LEFT_ID],
+		 off[SENSOR_FRONT_RIGHT_ID], on[SENSOR_SIDE_LEFT_ID],
+		 on[SENSOR_SIDE_RIGHT_ID], on[SENSOR_FRONT_LEFT_ID],
+		 on[SENSOR_FRONT_RIGHT_ID]);
+}
+
+/**
+ * @brief Log front sensors distances and error.
+ */
+void log_front_sensors_error(void)
+{
+	float sensors_error = get_front_sensors_error();
+	float fl_dist = get_front_left_distance();
+	float fr_dist = get_front_right_distance();
+
+	LOG_INFO("front_sensors_error:,%f,distances:%f,%f", sensors_error,
+		 fl_dist, fr_dist);
```

Comment:
This is not easily parseable. Either use "%f,%f,%f" or, if you want to include the variable names, then use something like "{"name":%f,...}" (see the `control_variables` above).